dev/com.ibm.ws.beanvalidation.v20.cdi/resources/com/ibm/ws/beanvalidation/v20/cdi/internal/BVNLSMessages.nlsprops

```diff
@@ -0,0 +1,46 @@
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# DO NOT TRANSLATE "\n" and "\t" characters.  these are for textual formatting.
+# REMINDER: if message has substitution parameters, then any apostrophe in the message text
+#           must be followed by a 2nd apostrophe to work correctly.
+# -------------------------------------------------------------------------------------------------
+#
+# Briefly:
+#     MESSAGE_ID=<ComponentPrefix><MessageNumber><TypeCode>: <MessageText>
+#
+# Message numbers are four digit numbers, from '0000' to '9999', inclusive.
+# Message numbers are filled to the left with '0', to a width of 4.
+#
+# Valid message type codes are:
+#     I (Informational)
+#     W (Warning)
+#     E (Error)
+#
+# Note the ": " that appears between the type code and the message text.
+#
+# -----------------------------------------------------------------------
+# cmvc_en_version_level=
+#CMVCPATHNAME ws/code/beanvalidation/src/com/ibm/ws/beanvalidation/resources/nls/BVNLSMessages.nlsprops
+#COMPONENTPREFIX CWNBV
+#COMPONENTNAMEFOR CWNBV Bean Validation
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_VAR
+
+# -----------------------------------------------------------------------
+# PARAMETERS:
+#    {0} the application name
+#    {1} the exception message from trying to register the CDI extension
+# -----------------------------------------------------------------------
+UNABLE_TO_REGISTER_WITH_CDI=CWNBV0200W: Unable to register Hibernate Validator CDI extension for application {0}. As a result, CDI integration for Bean Validation will not be available for this application. The issue registering the CDI extension was: {1}
+UNABLE_TO_REGISTER_WITH_CDI.explanation=A CDI extension is provided for the built-in Bean Validation provider (Hibernate Validator), which is only supported when the built-in ValidationProvider is used, and the validation.xml for the application is valid.  
+UNABLE_TO_REGISTER_WITH_CDI.useraction=See the root issue at the end of the message for a problem indication.  Likely causes include invalid content in META-INF/validation.xml or a <default-provider> other than org.hibernate.validator.HibernateValidator
```

Comment:
Sentence should end with period.  The phrase "for a problem indication" seem awkward.  It also seems redundant, so could possibly omit it rather than replacing with something else.